sphinx_compas_theme/compaspkg/layout.html

```diff
@@ -231,6 +240,20 @@ <h4 class="pb-3 footer-text">COMPAS</h4>
     }).addClass("active");
 
   });
+
+
+  let url = $("#doc_versions").data("url");
+  let docs_site = $("#doc_versions").data("docs");
+
+  if (url) {
+    $.get(url, res=>{
+      let lines = res.match(/[^\r\n]+/g);
```

Comment:
Splitting lines using regex is a bit overkill, you could do `res.split('\r\n')` instead.

Reply:
This was to at the same time deal with the default unix line break which is only `\n`. Unless this is not a valid concern?